go.mod

```diff
@@ -45,6 +45,7 @@ require (
 	github.com/ziutek/mymysql v1.5.4 // indirect
 	go.uber.org/zap v1.10.0
 	golang.org/x/tools v0.0.0-20191018212557-ed542cd5b28a
+	google.golang.org/genproto v0.0.0-20181016170114-94acd270e44e // indirect
```

Comment:
Still one change in the go.mod. Any particular reason for we add any module here?

Reply:
This was unintended, I have only committed watches.go file, not sure why go.mod is reflecting.

Reply:
solved